plugins/modules/source_control/gitlab/gitlab_group_members.py

```diff
@@ -185,6 +186,9 @@ def get_group_id(self, gitlab_group):
                 return group.id
         for group in groups:
             if group.path == gitlab_group or group.name == gitlab_group:
+                self._module.deprecate(
+                    msg="Setting 'gitlab_group' to 'name' or 'path' is deprecated. Use 'full_path' instead.",
```

Comment:
```suggestion
                    msg="Setting 'gitlab_group' to 'name' or 'path' is deprecated. Use 'full_path' instead",
```
Ah sorry, missed that earlier. Ansible adds a period at the end of the message, so when you also have one you end up with two... :)